app/full_text_search.py

```diff
@@ -204,7 +208,7 @@ def search(repo, query_txt, max_results):
 
     results_list = [non_romanized_person_location_index_results,
                     person_location_index_results]
-    index_results = get_person_ids_from_results(query_txt, results_list)
+    index_results = get_person_ids_from_results(query_txt, results_list, returned_fields)
```

Comment:
Can you pass returned_name_fields here instead of returned_fields? We don't want to include 'record_id' here.